e2e/scenario_test.go

```diff
@@ -190,7 +190,6 @@ func Test_AzureLinuxV2_GPU(t *testing.T) {
 				vmss.SKU.Name = to.Ptr("Standard_NC6s_v3")
 			},
 			Validator: func(ctx context.Context, s *Scenario) {
-				ValidatePodUsingNVidiaGPU(ctx, s)
```

Comment:
why are we removing all GPU tests from all SKUs expect for 1804 ? when 1804 will be EOL in 1 month.

Reply:
The validator I added is fragile, so I'm reverting the change in removing it from everywhere.

Just missed a comment for 1804. Removed.